feat: support api v10 (#353)
DisnakeDev/disnake: docs/intents.rst

This is a quick checklist to see if you need specific privileged intents.

.. _need_message_content_intent:

Message Content Intent
++++++++++++++++++++++++

- Whether you want a prefix that isn't the bot mention.

> - Whether you want to see what the content of a message was. This includes content, embeds, attachments, and components.

Reviewer: ```suggestion
- Whether you want to access the contents of messages. This includes content (text), embeds, attachments, and components.
```
first change isn't all that important, but clarifying what "content" actually is (to people who never worked with this) would be nice

Author: https://github.com/DisnakeDev/disnake/pull/353/commits/3c3fe5b2ad1b26b9bd0a23afb7462b05ccfd9058
